lib/record_store/zone.rb

```diff
@@ -32,10 +32,10 @@ def download(name, provider_name, **write_options)
         zone.write(**write_options)
       end
 
-      def filter_records(current_records, ignore_patterns)
-        ignore_patterns.inject(current_records) do |remaining_records, pattern|
+      def filter_records(current_records, ignore_objects)
+        ignore_objects.inject(current_records) do |remaining_records, pattern|
```

Comment:
What’s in a name? If the collection is objects instead of patterns, why are the individual items being enumerated still `pattern`?

Reply:
This is a holdover from a previous iteration where regexp based ignore patterns had a different type. I later took Angelo's advice and just added a field, so this change is unnecessary. Will revert this hunk.